### DISCUSSION TITLE
Windows 10 Search Box

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#2557] [Ideas]
Hi!
It would be very cool if your program would gav have the Windows 10 Search Box.
I'd appreciate it to do that :D
erschrecken

**Comment by pyrates999** [comment]
We don't have a method yet to resurrect the Windows 10 search, and we have no plans of bringing it back.

according to this: https://github.com/valinet/ExplorerPatcher/discussions/2507

**Comment by milestprower92** [comment]
No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the old search box from the old taskbar and there are no plans to have it added back due to the fact that the search icon does the same thing as the search box.
